ext/ftp/php_ftp.c

```diff
@@ -190,13 +233,11 @@ PHP_FUNCTION(ftp_login)
 	char *user, *pass;
 	size_t user_len, pass_len;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "rss", &z_ftp, &user, &user_len, &pass, &pass_len) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "Oss", &z_ftp, php_ftp_ce, &user, &user_len, &pass, &pass_len) == FAILURE) {
```

Comment:
The function signatures can also be changed accordingly in the stub file

Reply:
Yeah, noticed that after push, I've updated the commit already.